src/components/CarTypeBar/CarTypeBar.scss

```diff
@@ -0,0 +1,33 @@
+.carTypeBar {
+  padding-top: 0.5rem;
+  justify-content: center;
+  gap: 0.1rem;
+  height: 4rem;
+  background-color: black;
+  color: #9e9e9e;
```

Comment:
if these colors are part of the theme being used in designs these should be coming from proper theme object/file.